SonarQube.Common/AnalysisConfig/AnalysisConfig.cs

```diff
@@ -33,6 +33,12 @@ public class AnalysisConfig
         ///  See https://jira.sonarsource.com/browse/SONARMSBRU-100 for details.</remarks>
         public string SonarRunnerWorkingDirectory { get; set; }
```

Comment:
I think we can now also set the sonar-scanner working directory to the sources dir. This is already done for the command line option. See https://jira.sonarsource.com/browse/SONARMSBRU-100 for more details.